woof-code/rootfs-skeleton/etc/profile

```diff
@@ -2,13 +2,41 @@
 #100622 PKG_CONFIG_PATH is determined in 3builddistro (Woof), written to /etc/profile.d/pkgconfig
 #110804 fix double-login when exit from X, allow /etc/profile to complete. see also "echo -n '# '" in /usr/bin/xwin at exit.
 
-PATH="/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/root/my-applications/bin:/usr/games"
-if [ -f /lib64/libc.so.6 ] ; then #slackware64
-	LD_LIBRARY_PATH="/lib64:/usr/lib64:/root/my-applications/lib:/usr/local/lib"
-else
-	LD_LIBRARY_PATH="/lib:/usr/lib:/root/my-applications/lib:/usr/local/lib"
+[ "$PATH" == "" ] && export PATH="/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/usr/X11R7/bin"
+
+if [ "$LD_LIBRARY_PATH" == "" ]; then
```

Comment:
LD_LIBRARY_PATH is an alternative to /etc/ld.so.conf. Why not just rely on /etc/ld.so.conf, like other distros?